2023-06-02-Malawi-MSF-Cholera-Cyclone-Response.md

```diff
@@ -0,0 +1,124 @@
+---
+layout: post
+title: "Remote Mapping in Malawi: Supporting Field Operations"
+postID: 2023-06-02-Malawi-MSF-Cholera-and-Cyclone-Freddy-Response
+category: blog
+banner: https://www.missingmaps.org/img/images/missingmaps-blog_20230601_malawi_banner.jpeg
+date: 2023-06-02
+author: Jana Bauerová, Sophie Gélinas-Gagné and Alexis Huart
+excerpt: "Malawi was experiencing the worst cholera epidemic in 20 years, so MSF sent an emergency team and activated Missing Maps. When a cyclone hit in March, OSM data was also invaluable for assessment."
+published: true
+tags: [MSF, Medecins Sans Frontieres, Health, Cholera, Malawi, OpenStreetMap, Cyclone]
+permalink: /blog/:year/:month/:day/:title/
+lang: en
+---
+
+---
+  
+# Remote Mapping in Malawi: Supporting Field Operations for the Cholera Outbreak and Cyclone Freddy Response
```

Comment:
the `title` from the metadata will display inside the image header on the blog post page. including it again here will repeat it. suggest removing if not intentional.

Reply:
intentional